doc/getting-started/overview_dev.rst

```diff
@@ -3,19 +3,22 @@
 Configuration and Development Overview
 ######################################
 
-This overview is for developers who are new or relatively new to ACRN. It will
-help you get familiar with ACRN basics: ACRN components and general process for
-building an ACRN hypervisor.
+This overview is for developers who are new or relatively new to ACRN and are
+responsible for configuring and building ACRN hypervisors. It will introduce you
+to the general development process, including ACRN components and tools.
 
 The overview covers the process at an abstract and universal level.
 
 * Abstract: the overall structure rather than detailed instructions
 * Universal: applicable to most use cases
 
-Although the overview describes the process as a series of steps, it's intended
-to be a summary, not a step-by-step guide. Throughout the overview, you will see
-links to the :ref:`gsg` for first-time setup instructions. Links to advanced
-guides and additional information are also provided.
+The overview is intended to complement the :ref:`gsg`. The guide provides
+step-by-step instructions to enable an ACRN example for first-time use, while
+the overview provides background information and serves as a gateway to
+additional features and resources that can help you develop your solution.
+
+The overview doesn't cover ACRN benefits, use cases, or architecture. See
```

Comment:
How about a positive statement instead (drop the first sentence, "The overview doesn't..."):
```
See :ref:`introduction` for information about ACRN benefits, use cases, and architecture.
```